cmd/cri-containerd/cri_containerd.go

```diff
@@ -82,9 +82,29 @@ func versionCommand() *cobra.Command {
 
 func loadImageCommand() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "load TAR",
+		Use: "load TAR",
+		Long: `Help for "load TAR" command
+
+TAR - The path to a tar archive containing a container image.
+
+Requirement:
+	Containerd and cri-containerd daemons (grpc servers) must already be up and
+	running before the load command is used.
+
+Description:
+	Running as a client, cri-containerd implements the load command by sending the
+	load request to the already running cri-containerd daemon/server, which in
+	turn loads the image into containerd's image storage via the already running
+	containerd daemon.`,
 		Short: "Load an image from a tar archive.",
 		Args:  cobra.ExactArgs(1),
+		Example: `
+	Example 1
```

Comment:
Given the output, I feel like `Example 1` is not that necessary. :)

Reply:
And could you use `dedent` for these strings? https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/completion.go#L66

Reply:
well it says Examples by default not sure where to remove that...

Reply:
I think we could just use the `Example` it prints. :)

Reply:
The example it prints is plural `Examples:`

Reply:
decent() any particular reason for that?

Reply:
Let me try and tell you.

Reply:
Here is the output with dedent:
```
$ ./cri-containerd help load

test
    infra

Usage:
  cri-containerd load TAR [flags]

Flags:
      --endpoint string    cri-containerd endpoint. (default "/var/run/cri-containerd.sock")
  -h, --help               help for load
      --timeout duration   cri-containerd connection timeout. (default 10s)

Global Flags:
      --alsologtostderr                  log to standard error as well as files
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --logtostderr                      log to standard error instead of files
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          log level for V logs
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging
```

Here is the output without dedent:
```
$ ./cri-containerd help load

		test
		    infra

Usage:
  cri-containerd load TAR [flags]

Flags:
      --endpoint string    cri-containerd endpoint. (default "/var/run/cri-containerd.sock")
  -h, --help               help for load
      --timeout duration   cri-containerd connection timeout. (default 10s)

Global Flags:
      --alsologtostderr                  log to standard error as well as files
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --logtostderr                      log to standard error instead of files
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          log level for V logs
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging
```

And here is the code:
```
                Short: "Load an image from a tar archive.",
-               Args:  cobra.ExactArgs(1),
+               Long: dedent.Dedent(`
+               test
+                   infra
+               `),
+               Args: cobra.ExactArgs(1),
        }
```

Reply:
It just make us be able to indent multi-line string in code properly.

Reply:
nvm.. done...

Reply:
I replaced the Examples text in the template..